### ISSUE TITLE
Build failing on windows with link error for accessibility_cc.pdb

### ISSUE THREAD

**Issue by nripendra** [electron/electron#8833]
It gives a lot of warning for platform_cc.pdb and generated_cc.pdb, but finally fails with error for accessibility_cc.pdb. Why are these pdb files missing from the build?

**Failure info**
webkitmodules.lib(InspectorTypeBuilderHelper.obj) : warning LNK4099: PDB 'accessibility_cc.pdb' was not found with 'webkitmodules.lib(InspectorTypeBuilderHelper.obj)' or at 'D:\Projects\electron\out\R\accessibility_cc.pdb'; linking object as if no debug info
**webrtc.lib(app.obj) : fatal error LNK1318: Unexpected PDB error; NOT_FOUND (4) ' D:\Projects\electron\out\R\accessibility_cc.pdb'**
Traceback (most recent call last):
  File "gyp-win-tool", line 323, in <module>
    sys.exit(main(sys.argv[1:]))
  File "gyp-win-tool", line 29, in main
    exit_code = executor.Dispatch(args)
  File "gyp-win-tool", line 71, in Dispatch
    return getattr(self, method)(*args[1:])
  File "gyp-win-tool", line 179, in ExecLinkWithManifests
    subprocess.check_call(ldcmd + add_to_ld)
  File "E:\portable-softwares\depot_tools\python276_bin\lib\subprocess.py", line
 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'E:\portable-softwares\depot_tools\python
276_bin\python.exe gyp-win-tool link-wrapper environment.x64 False link.exe /nol
ogo /OUT:electron.exe @electron.exe.rsp electron.exe.manifest.res' returned non-
zero exit status 1318
ninja: build stopped: subcommand failed.

**Comment by bsclifton** [comment]
@nripendra are you out of disk space? I've commonly gotten that type of error when I'm running low

The pdb is built during the build

**Comment by nripendra** [comment]
@bsclifton no, I have 73 gb free space in the drive in which I'm building this project. I've tried deleting the `out` folder and re-run script/update, script/build with no success.

**Comment by cifratila** [comment]
@nripendra What are your steps for building electron?

**Comment by nripendra** [comment]
@cifratila 

- clone electron, then executed script/bootstrap, script/update and then script/build

**Comment by cifratila** [comment]
@nripendra Maybe I might be missing something, but why do you run update? It is already called by bootstrap.

For me this flow works just fine:
python script/bootstrap.py -v
python script/build.py -c D

**Comment by nripendra** [comment]
@cifratila 

This worked for be too. Not sure why release build doesn't work?

I'm still getting following warning, but now electron.exe is getting built.
```
ninja: Entering directory `out\D'
[1233/1233] LINK_EMBED electron.exe
libyuv.lib(jsimd_x86_64.obj) : warning LNK4099: PDB 'simd_c.pdb' was not found with 'libyuv.lib(jsimd_x86_64.obj)' or at 'D:\Projects\electron\out\D\simd_c.pdb'; linking object as if no debug info
```

**Comment by cifratila** [comment]
@nripendra 

Yeah I get the same warning :) That will probably need more investigation. Didn't try to debug something to see if it actually is breaking anything, but apps work with this build.

**Comment by kevinsawicki** [comment]
> libyuv.lib(jsimd_x86_64.obj) : warning LNK4099: PDB 'simd_c.pdb' was not found with 'libyuv.lib(jsimd_x86_64.obj)' or at 'D:\Projects\electron\out\D\simd_c.pdb'; linking object as if no debug info

This was fixed in https://github.com/electron/libchromiumcontent/pull/262

**Comment by kevinsawicki** [comment]
@nripendra are you still experience this or can it be closed out?

**Comment by nripendra** [comment]
Yes, please this can be closed
